examples/AdminClient/Program.cs

```diff
@@ -403,6 +428,15 @@ static void PrintListOffsetsResultInfos(List<ListOffsetsResultInfo> ListOffsetsR
             }
         }
 
+        static void PrintElectLeaderResults(ElectLeadersResult result)
+        {
+            Console.WriteLine("  ElectLeadersResult:");
+            foreach (var partitionResult in result.PartitionResults)
+            {
+                Console.WriteLine($"Election successful in {partitionResult.Topic} {partitionResult.Partition}");
+            }
+        }
```

Comment:
This can be common for the Result and Report by passing the partition list.
```suggestion
        static void PrintElectLeaderResults(List<TopicPartitionError> topicPartitions)
        {
            Console.WriteLine($"ElectLeaders response has {topicPartitions.Count} partition(s):");
            foreach (var partitionResult in topicPartitions)
            {
                if (!partitionResult.Error.IsError)
                    Console.WriteLine($"Election successful in {partitionResult.Topic} {partitionResult.Partition}");
                else
                    Console.WriteLine($"Election failed in {partitionResult.Topic} {partitionResult.Partition}: " +
                      $"Code: {partitionResult.Error.Code}" +
                      $", Reason: {partitionResult.Error.Reason}");
            }
        }
```